src/main/java/uk/gov/ons/census/fwmt/jobservice/service/CeFollowUpSchedulingService.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ons.census.fwmt.jobservice.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+/**
+ * This class is a caching service that reduces the frequency of date checks
+ */
+
+@Slf4j
+@Service
+public class CeFollowUpSchedulingService {
+  @Value("${spg.followUpDate}")
+  Long followUpDate;
+  private boolean inFollowUp = false;
```

Comment:
Add another field Date startDate from a field in application.yml 
ce.startdate

Reply:
This has been done.